Nuro/Utilities/CoreData/Repository/RoutineHeaderLocalRepository.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  RoutineHeaderLocalRepository.swift
+//  Nuro
+//
+//  Created by Karen Natalia on 13/10/22.
+//
+
+import Foundation
+import CoreData
+
+protocol RoutineHeaderRepository {
+    func add(dayID: Int, timeID: Int)
+    func getAll() -> [RoutineHeader]
+    func getRoutinesOfDay(dayID: Int) -> [RoutineHeader]
+    func getRoutineOfDayAndTime(dayID: Int, timeID: Int) -> RoutineHeader
+}
```

Comment:
_Discussion_: Why not extract all CoreData protocols to another file?
`ex: CoreDataProtocols`